src/SourceLocator/Ast/FindReflectionsInTree.php

```diff
@@ -96,6 +97,37 @@ public function enterNode(Node $node)
                     return null;
                 }
 
+                if ($node instanceof Node\Stmt\ClassConst) {
+                    return null;
+                }
+
+                if ($node instanceof Node\Stmt\Const_) {
+                    for ($i = 0; $i < count($node->consts); $i++) {
+                        $reflection = $this->astConversionStrategy->__invoke($this->reflector, $node, $this->locatedSource, $this->currentNamespace, $i);
+
+                        if (! $this->identifierType->isMatchingReflector($reflection)) {
+                            continue;
+                        }
+
+                        $this->reflections[] = $reflection;
+                    }
+
+                    return null;
+                }
+
+                if ($node instanceof Node\Expr\FuncCall
+                    && $node->name instanceof Node\Name
+                    && $node->name->toLowerString() === 'define'
```

Comment:
Do we operate on a FQN here? Risky otherwise, as `define()` can be declared in userland

Reply:
Changed the condition to ` && $node->name instanceof Node\Name\FullyQualified`

Reply:
Hmm, I cannot use `Node\Name\FullyQualified` because `define` call in namespace is resolved as `NamespaceName\define`. PHP itself will fallback to global `define` so I have to ignore FQN.

Reply:
This is still an issue IMO: we could do a lookup for `NamespaceName\define` and fall back to `\define` if it is not existing